ocs_ci/helpers/dr_helpers_ui.py

```
@@ -200,6 +204,197 @@ def verify_drpolicy_ui(acm_obj, scheduling_interval):
+        ).get_attribute("aria-expanded")
+
+        if critical_alert == "false":
+            critical_alert_elem = wait_for_element_to_be_clickable(
+                acm_loc["critical-alert"]
+            )
+            acm_obj.driver.execute_script("arguments[0].click();", critical_alert_elem)
+            acm_obj.take_screenshot()
+    except (NoSuchElementException, StaleElementReferenceException) as e:
+        log.warning(
+            f"Could not expand Critical alerts section: {e}. "
+            "Proceeding to verify alert presence directly."
```

Ruff (0.15.20)

[warning] Logging statement uses f-string

(G004)